examples/other/wifi-img-streamer/wifi-img-streamer.c

```diff
@@ -318,10 +318,10 @@ void camera_task(void *parameters)
 
         transferTime = xTaskGetTickCount() - start;
       }
-#ifdef OUTPUT_PROFILING_DATA
+// #ifdef OUTPUT_PROFILING_DATA
```

Comment:
I would probably remove the comments on the #ifdef here. Or else there is a looot of printouts on the console 😅

Reply:
true, that was not intended to be still in there. I changed it.